go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/throttled/throttled
+
+go 1.14
+
+require (
+	github.com/go-redis/redis v6.15.8+incompatible
+	github.com/gomodule/redigo v1.8.2
+	github.com/hashicorp/golang-lru v0.5.4
+	github.com/onsi/ginkgo v1.13.0 // indirect
+)
```

Comment:
Would you mind stripping the `go.*` files out of the pull request for now? I'll bring Go Modules support in separately.